docs/src/FAQ.md

```diff
@@ -72,7 +72,7 @@ You can of course in turn build and deploy this package with
 julia build_tarballs.jl --debug --verbose --deploy="MY_USERNAME/Quxlib_jll.jl"
 ```
 
-Note that `PackageSpec` can also point to a local path: e.g., `PackageSpec(; name="Libfoo_jll", uuid="...", path="/home/myname/.julia/dev/Libfoo_jll")`.  This is particularly useful when [building a custom JLL package locally](@ref), instead of deploying it to a remote Git repository.
+Note that `PackageSpec` can also point to a local path: e.g., `PackageSpec(; name="Libfoo_jll", uuid="...", path="/home/myname/.julia/dev/Libfoo_jll")`.  This is particularly useful when [building a custom JLL package locally](https://docs.binarybuilder.org/stable/building/#Building-a-custom-JLL-package-locally), instead of deploying it to a remote Git repository.
```

Comment:
It should work with `@ref` if you make it match the header name, so probably just need to update `b` -> `B` in `building`.

Reply:
Ok I see. Done.